quickwit-indexing/src/actors/garbage_collector.rs

```diff
@@ -52,6 +52,9 @@ pub struct GarbageCollectorCounters {
     pub num_deleted_bytes: usize,
 }
 
+#[derive(Debug)]
```

Comment:
Maybe this could be exported from actor crate as a dedicated utility

Reply:
The loop message you mean? That might make sesnse but I actually like it to be private to the actor.